dlp/snippets/src/test/java/dlp/snippets/JobsTests.java

```diff
@@ -31,41 +31,49 @@
 import com.google.privacy.dlp.v2.StorageConfig;
 import java.io.IOException;
 import java.util.UUID;
+import org.junit.BeforeClass;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
 @RunWith(JUnit4.class)
 public class JobsTests extends TestBase {
 
+  private static DlpServiceClient dlpServiceClient;
+
   @Override
   protected ImmutableList<String> requiredEnvVars() {
     return ImmutableList.of("GOOGLE_APPLICATION_CREDENTIALS", "GOOGLE_CLOUD_PROJECT", "GCS_PATH");
   }
 
+  @BeforeClass
+  public static void setUp() throws Exception {
+    // Initialize the Dlp Service Client.
+    dlpServiceClient = DlpServiceClient.create();
+  }
+
   private static DlpJob createJob(String jobId) throws IOException {
```

Comment:
The same change is made in this PR as well: https://github.com/GoogleCloudPlatform/java-docs-samples/pull/7922/

You might have to resolve conflicts in either of the PRs depending on which is merged first.

Reply:
Yeah, sure